docs/SYNTAX.md

```diff
@@ -98,17 +115,24 @@ x = phexByteStr "41"
 ```
 
 # Lambdas
+
 You can create Plutarch level lambdas by apply `plam` over a Haskell level lambda/function.
```

Comment:
I think it may be useful here to highlight that `plam` works "over a Haskell level lamba/function _of Plutarch terms_".  I'm re-reading this for the first time, and stumbled again thinking that I could just yeet `plam` in front of any Haskell function, for a second.